### PR TITLE
[Docs] Add missing php opening tag

### PR DESCRIPTION
**Pull request by dobryy**
This is an easy fix. We have one more rule with extra file content and it has opening php tag.

 We might add PHPStan rule that will check that this tag is present or... make doc generator check whether it is present or not. If not it can add it.

### CODE REVIEW

File: phpstan.neon [hunk 1]
```diff
@@ -10,7 +10,7 @@ includes:
 services:
     # require constant in argument position
     -
-        class: Symplify\CodingStandard\Rules\ForceMethodCallArgumentConstantRule
+        class: Symplify\CodingStandard\Rules\RequireMethodCallArgumentConstantRule
```
Comment (dobryy) [inline phpstan.neon:13]: Sneaky @TomasVotruba :D

File: packages/better-php-doc-parser/src/PhpDocInfo/PhpDocInfoFactory.php [hunk 2]
```diff
@@ -90,7 +90,8 @@ public function createFromNode(Node $node): ?PhpDocInfo
             $tokens = [];
             $phpDocNode = new AttributeAwarePhpDocNode([]);
         } else {
-            $content = $node->getDocComment()->getText();
+            $content = $node->getDocComment()
+                ->getText();
```
Comment (dobryy) [inline packages/better-php-doc-parser/src/PhpDocInfo/PhpDocInfoFactory.php:94]: @TomasVotruba , these kind of fixes by `fix-cs` look weird to me... I don't like :)
I could agree that it makes sense when the line is too long, or to make line break after the second chained method. :)

File: packages/better-php-doc-parser/src/PhpDocNodeFactory/MultiPhpDocNodeFactory.php [hunk 3]
```diff
@@ -61,16 +61,13 @@ public function getTagValueNodeClassesToAnnotationClasses(): array
             IdTagValueNode::class => 'Doctrine\ORM\Mapping\Id',
             GeneratedValueTagValueNode::class => 'Doctrine\ORM\Mapping\GeneratedValue',
             JoinColumnTagValueNode::class => 'Doctrine\ORM\Mapping\JoinColumn',
-
```
Comment (dobryy) [inline packages/better-php-doc-parser/src/PhpDocNodeFactory/MultiPhpDocNodeFactory.php:64]: @TomasVotruba removing one empty line also doesn't make sense :) It makes sense to replace multiple empty lines by one. Sometimes it is handy to group lines somehow.

Comment (TomasVotruba) [reply packages/better-php-doc-parser/src/PhpDocNodeFactory/MultiPhpDocNodeFactory.php:64]: What ECS rule is causing this?

File: phpstan.neon (hunk 1 above)
Comment (TomasVotruba) [reply phpstan.neon:13]: This should be fixed in `master`. Have you rebased? :)

File: packages/better-php-doc-parser/src/PhpDocInfo/PhpDocInfoFactory.php (hunk 2 above)
Comment (TomasVotruba) [reply packages/better-php-doc-parser/src/PhpDocInfo/PhpDocInfoFactory.php]: I see. The rule of 1 method chain call per line is more consistent and logical to read.

Same way 1 item per array item line.

```php
[
    'one' => 'two',
    'one' => 'two',
    'one' => 'two',
]
```

File: packages/better-php-doc-parser/src/PhpDocNodeFactory/MultiPhpDocNodeFactory.php (hunk 3 above)
Comment (dobryy) [reply packages/better-php-doc-parser/src/PhpDocNodeFactory/MultiPhpDocNodeFactory.php]: I didn't notice it. We will find it later

**Comment by dobryy** [comment]
@TomasVotruba Ready for merging

**Pull request merged**